website/docs/r/compute_project_metadata.html.markdown

```diff
@@ -8,7 +8,10 @@ description: |-
 
 # google\_compute\_project\_metadata
 
-Manages metadata common to all instances for a project in GCE.
+Manages metadata common to all instances for a project in GCE. If you
```

Comment:
I almost documented exactly this, but unfortunately there's a bug right now in how `google_compute_project_metadata` works; right now it works similar to this PR, except upon delete (which deletes the entire map). We have a work item to go fix this so that it manages the entire metadata, but until then I'm hesitant to document that behavior (hence the current carefully worded description).